README.md

```diff
@@ -65,6 +60,11 @@ Note that non-default networks require extra [firewall setup](https://cloud.goog
   | Immediate            |       N/A         |        Present      | Call CreateVolume with requisite set to allowedTopology and preferred set to the sorted and shifted version of requisite at a randomized index |
   | Immediate            |       N/A         |        Not Present  | Call CreateVolume with requisite = aggregated topology across nodes which contain the topology keys of CSINode objects, preferred = sort and shift requisite at a randomized index |
 
+* Filestore Backups: Filestore Backup relies on CSI VolumeSnapshot which is a Beta feature in k8s enabled by default in
+Kubernetes 1.17+. CSI VolumeSnapshot should not be confused with Filestore Backups. Filestore CSI driver leverages CSI VolumeSnapshot capability to support Filestore Backups by specifying a `type` parameter in the VolumeSnapshotClass object. For more details see user-guide [here](docs/kubernetes/backup.md).
```

Comment:
Filestore snapshots is a publicly announced alpha feature: https://cloud.google.com/sdk/gcloud/reference/alpha/filestore/snapshots/create

I think the confusion is going to be with CSI snapshots vs filestore snapshots vs backups. 

Since the rest of this document is in terms of CSI/k8s features, maybe the bullet here should be from a CSI perspective instead of a filestore one. Namely, something like:

* Volume Snapshots: This driver currently supports CSI VolumeSnapshots on a Filestore instance using the GCP Filestore Backup feature. CSI VolumeSnapshot is a Beta feature in k8s enabled by default in 1.17+. The GCP [Filestore Snapshot alpha](https://cloud.google.com/sdk/gcloud/reference/alpha/filestore/snapshots/create) is not currently supported, but will be in the future via the `type` parameter in the VolumeSnapshotClass. For more details see the user-guide [here](docs/kubernetes/backup.md).

Reply:
Done